instrumentation/lettuce/lettuce-5.1/src/main/java/io/opentelemetry/javaagent/instrumentation/lettuce/v5_1/OpenTelemetryTracing.java

```diff
@@ -36,6 +36,9 @@
   public static final io.opentelemetry.trace.Tracer TRACER =
       OpenTelemetry.getGlobalTracer("io.opentelemetry.auto.lettuce-5.1");
 
+  private static final RedisCommandNormalizer COMMAND_NORMALIZER =
+      new RedisCommandNormalizer("lettuce", "lettuce-5", "lettuce-5.1");
```

Comment:
I don't like repeating the instrumentation names here, but you can't exactly access the `Instrumenter` from the advice. Maybe it'll be possible to fix that once https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/1274#issuecomment-707860035 is implemented.

Reply:
yes, that would be very nice to tie this to the existing instrumentation names 👍